### ISSUE TITLE
Module not found error in Jupyter notebook while open3d can be seen installed.. Is Open3D not supported on python versions above3.9 ?

### ISSUE THREAD

**Issue by Inshu32** [isl-org/Open3D#6420]
### Discussed in https://github.com/isl-org/Open3D/discussions/6417

<div type='discussions-op-text'>

<sup>Originally posted by **Inshu32** October  7, 2023</sup>
I installed open3d in base and my virtual environment, but when I open Jupyter notebook through my Linux terminal (debian based) and try to import open3d it says 'Module not found*. I installed it by building from source so that I have correct dependencies. But still snot working. I am working with tensorflow and Keras CV in the same virtual environment, if that messes up any thing ? I tried both kernels  - python as well as my virtual environment in jupyter notebook. Still its the same error. </div>

**Comment by saurabheights** [comment]
@Inshu32 Replied in https://github.com/isl-org/Open3D/discussions/6417. Also, please try to reproduce in a fresh conda environment and if the error still happens, share the steps. Also, please follow [new github issue template](https://github.com/isl-org/Open3D/issues/new?assignees=&labels=build%2Finstall+issue&projects=&template=0-build-issue-report.yml&title=Summarize+the+issue+and+your+environments+%28e.g.%2C+%22Cannot+compile+on+Ubuntu+20.04+with+OpenBLAS%22%29) when reporting issue.
